README.md

```diff
@@ -45,10 +51,34 @@ Run the operator:
 $ make run
 ```
 
+Clean up the Kubernetes/OpenShift cluster:
+```
+$ make cluster/clean
+```
+
 ## Via the Operator Catalog
 
 ***In development***
 
+## Usage
+### Provider
+A config map object is expected to exist with a mapping from type name to deployment method, and example of this can be seen [here](./deploy/examples/config_aws_provider.yaml)
```

Comment:
Could we rename `config_aws_provider.yaml` to `operator_config.yaml` (may also need to be changed in the Makefile). I don't think the name is accurate anymore